packages/rpc-graphql/README.md

```diff
@@ -3,61 +3,1160 @@
+            owner: {
+                address: 'TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA',
+                owner: {
+                    address: 'BPFLoader2111111111111111111111111111111111',
+                    owner: {
+                        address: 'NativeLoader1111111111111111111111111111111',
+                        owner: {
+                            address: '11111111111111111111111111111111',
+                            executable: true,
+                        },
+                    },
+                },
+            },
+        },
+    },
+});
+```
+
+Nested queries can also be applied to specific account types.
+
+```typescript
+const source = `
+    query myQuery($address: String!) {
+        account(address: $address) {
+            ... on MintAccount {
+                address
+                data {
+                    parsed {
+                        info {
+                            mintAuthority {
+                                address
+                                lamports
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+`;
+
+const variableValues = {
+    address: 'AyGCwnwxQMCqaU4ixReHt8h5W4dwmxU7eM3BEQBdWVca',
+};
+
+const result = await rpcGraphQL.query(source, variableValues);
+
+expect(result).toMatchObject({
+    data: {
+        account: {
+            address: 'AyGCwnwxQMCqaU4ixReHt8h5W4dwmxU7eM3BEQBdWVca',
+            data: {
+                parsed: {
+                    info: {
+                        mintAuthority: {
+                            address: expect.any(String),
+                            lamports: expect.any(BigInt),
+                        }
+                    }
+                }
+            },
+        },
+    },
+});
+```
+
+## Querying Program Accounts
+
+A very common way to query Solana accounts from an RPC is to request all of
+the accounts owned by a particular program.
+
+With RPC-GraphQL, querying program-owned accounts is a list-based extension of
+the account query defined previously. This means program accounts queries will
+return a list of objects implementing the `Account` interface.
+
+Setting up the query is very similar to the `account` query.
+
+```typescript
+const source = `
+    query myQuery($programAddress: String!) {
+        programAccounts(programAddress: $address) {
+            executable
+            lamports
+            rentEpoch
+        }
+    }
+`;
+
+const variableValues = {
+    programAddress: 'AmtpVzo6H6qQCP9dH9wfu5hfa8kKaAFpTJ4aamPYR6V6',
+};
+
+const result = await rpcGraphQL.query(source, variableValues);
+
+expect(result).toMatchObject({
+    data: {
+        // Array of `Account` objects
+        programAccounts: expect.arrayContaining([{
+            executable: expect.any(Boolean),
+            lamports: expect.any(BigInt),
+            rentEpoch: expect.any(BigInt),
+        }])
+    },
+});
```

Comment:
Nitpick: It might be clearer to show an example output here, either instead of or in addition to the `expect.any` stuff

Reply:
You mean like a console log? Or a type?

Reply:
I guess a console log, I just think seeing the actual output of some of these is more impactful/clear than the `expect` shape

Reply:
Added!